platform/lib/platform/material/media.ex

```diff
@@ -115,6 +117,10 @@ defmodule Platform.Material.Media do
       user: user,
       required: true
     )
+    |> Attribute.validate_attribute(Attribute.get_attribute(:geolocation), media,
+      user: user,
+      required: true
```

Comment:
Hmm — why should geolocation be required when creating an incident?

Reply:
Yes, it is not required, I removed this, thanks!